drape_frontend/poi_symbol_shape.cpp

```diff
@@ -91,7 +91,7 @@ void SolidBatch(ref_ptr<dp::GraphicsContext> context, ref_ptr<dp::Batcher> batch
   state.SetTextureFilter(dp::TextureFilter::Nearest);
   state.SetTextureIndex(symbolRegion.GetTextureIndex());
 
-  dp::AttributeProvider provider(1 /* streamCount */, vertexes.size());
+  dp::AttributeProvider provider(1 /* streamCount */, (uint32_t)vertexes.size());
```

Comment:
I don't like this kind of suppressing. Better to leave as is or static_cast.

Reply:
Reverted.